### DISCUSSION TITLE
No Cover Art For Radio

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#530] [Fixed Issues]
I am not getting any cover art for webradio stations. Works OK for local sources.

rAudio 1 • 20210927
5.10.63-12-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2

**Comment by klassikmann** [comment]
Can't confirm, coverart for web radios o.k.

rAudio 1 • 20210927
5.10.63-14-ARCH aarch64
Raspberry Pi 3 Model B Rev 1.2

**Comment by R101G** [comment]
The Radio Paradise website is down this morning. When I tried another station, that was also not working, but it looks like a different problem, unless they also use Amazon servers.

Edit: This is clearly not an rAudio problem, but I cannot see a way to delete the thread so will move it to fixed.
